modules/core/src/test/java/org/apache/ignite/internal/IgniteDiscoveryMassiveNodeFailTest.java

```diff
@@ -301,15 +301,23 @@ public void testRecoveryOnDisconnect() throws Exception {
      *
      */
     private class FailDiscoverySpi extends TcpDiscoverySpi {
-        /** {@inheritDoc} */
-        @Override protected void writeToSocket(Socket sock, TcpDiscoveryAbstractMessage msg, byte[] data,
+        /**
+         * Writes message to the socket.
+         *
+         * @param sock Socket.
+         * @param msg Message.
```

Comment:
@aleksey-kurinov , why replaced the inherited doc?

Reply:
fixed. i've reviewd code obe more time and i've veririden method instead of overloading. so doc is inherited now.
TcpDiscoveryConnectionCheckMessage is used as a stub instead of msg which was pased via parameter according to your comment:
// Replace logic routine message with a stub to update last-sent-time to avoid segmentation on
// connRecoveryTimeout.